azure-pipelines.yml

```diff
@@ -90,7 +90,7 @@ stages:
         displayName: "Build NuGet packages"
         inputs:
           command: pack
-          packagesToPack: 'analyzers/src/**/*.nuspec'
+          packagesToPack: 'analyzers/src/**/*.nuspec;analyzers/packaging/**/*.nuspec'
```

Comment:
AFAICT there are no `nuspec` files in `src` now
```suggestion
          packagesToPack: 'analyzers/packaging/**/*.nuspec'
```

Reply:
This will skip the CFG package generation.

Reply:
Or like this
```suggestion
          packagesToPack: 'analyzers/src/**/*.nuspec;analyzers/packaging/*.nuspec'
```